dora/stress/shell/src/main/java/alluxio/stress/cli/worker/StressWorkerBench.java

```diff
@@ -381,8 +381,7 @@ public long getEndMs() {
 
     public synchronized void mergeThreadResult(WorkerBenchTaskResult threadResult) {
       if (mResult == null) {
-        mResult = threadResult;
-        return;
+        mResult = new WorkerBenchTaskResult();
```

Comment:
not using the input arg?

Reply:
Yes, it will create an empty TaskResult first if the mResult is null. Then for any thread result, they follow the same procedure to merge into mResult

Reply:
ah got it, i missed L389